src/main/java/seedu/address/model/person/Email.java

```diff
@@ -10,26 +10,20 @@
 public class Email {
 
     private static final String SPECIAL_CHARACTERS = "+_.-";
-    public static final String MESSAGE_CONSTRAINTS = "Emails should be of the format local-part@domain "
-            + "and adhere to the following constraints:\n"
-            + "1. The local-part should only contain alphanumeric characters and these special characters, excluding "
-            + "the parentheses, (" + SPECIAL_CHARACTERS + "). The local-part may not start or end with any special "
-            + "characters.\n"
-            + "2. This is followed by a '@' and then a domain name. The domain name is made up of domain labels "
-            + "separated by periods.\n"
+    public static final String MESSAGE_CONSTRAINTS = "Emails should be of the format local-part@domain.tld and adhere"
+            + "to the following constraints:\n"
+            + "1. The local-part should only contain alphanumeric characters and these special characters ("
+            + SPECIAL_CHARACTERS + "). The local-part may not start or end with any special characters.\n"
+            + "2. This is followed by a '@' , a domain name, a '.', and the top-level domain (tld). "
+            + "The domain name is made up of domain labels separated by periods.\n"
             + "The domain name must:\n"
-            + "    - end with a domain label at least 2 characters long\n"
+            + "    - end with a top-level domain (tld) which is at least 2 characters long\n"
```

Comment:
I'm a bit concerned that the `tld` abbreviation explanation is placed so low in the constraints message, since I do not expect our target audience would understand what `tld` means in the initial prompt at `Line 13`. Perhaps we can explain what this abbreviation early in the chunk?

Reply:
That's a very good point, I will move the explanation of this to a earlier part of the message.